tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noImplicitAny": true,
     "moduleResolution": "node",
     "baseUrl": "./",
+    "skipLibCheck": true,
```

Comment:
Why is this change required?

Reply:
👋 I created https://github.com/DataDog/datadog-ci/pull/1670 to unblock this PR